shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -542,17 +542,62 @@ protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight)
         super.onSizeChanged(width, height, oldWidth, oldHeight);
     }
 
+    // Decide if we want to zero the sides to remove padding for hidden navbar.
```

Comment:
Add a bit more detail. Specifically: 'zero the sides' -> 'zero the left/right-side padding of the window when the system navbar is hidden'.